vllm/model_executor/models/minicpm.py

```diff
@@ -163,10 +164,13 @@ def __init__(
                                            hidden_size,
                                            bias=False,
                                            quant_config=quant_config)
-        if hidden_act != "silu":
+        if hidden_act == "silu":
+            self.act_fn = SiluAndMul()
+        elif hidden_act == "fatrelu":
+            self.act_fn = FatreluAndMul(threshold=hidden_act_param)
+        else:
             raise ValueError(f"Unsupported activation: {hidden_act}. "
-                             "Only silu is supported for now.")
-        self.act_fn = SiluAndMul()
+                             "Only silu and fatrelu are supported for now.")
```

Comment:
Nit: I think it would be nice to use `get_act_fn()` and register this as an activation function https://github.com/vllm-project/vllm/blob/415f76a9cbcdec9346661e5b6f04c35a4d8eb3f4/vllm/model_executor/layers/activation.py#L220-L228 - however I recognize that this function requires the threshold to be piped through so it might not be worth it